Packages/Sites/Sfi.Kateheo/Resources/Private/Partials/SharedResources.html

```diff
@@ -17,22 +17,27 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
 
 		<!-- Generate favicons with http://realfavicongenerator.net/ -->
-		<link rel="apple-touch-icon" sizes="57x57" href="/apple-touch-icon-57x57.png">
-		<link rel="apple-touch-icon" sizes="114x114" href="/apple-touch-icon-114x114.png">
-		<link rel="apple-touch-icon" sizes="72x72" href="/apple-touch-icon-72x72.png">
-		<link rel="apple-touch-icon" sizes="144x144" href="/apple-touch-icon-144x144.png">
-		<link rel="apple-touch-icon" sizes="60x60" href="/apple-touch-icon-60x60.png">
-		<link rel="apple-touch-icon" sizes="120x120" href="/apple-touch-icon-120x120.png">
-		<link rel="apple-touch-icon" sizes="76x76" href="/apple-touch-icon-76x76.png">
-		<link rel="apple-touch-icon" sizes="152x152" href="/apple-touch-icon-152x152.png">
-		<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon-180x180.png">
-		<link rel="icon" type="image/png" href="/favicon-192x192.png" sizes="192x192">
-		<link rel="icon" type="image/png" href="/favicon-160x160.png" sizes="160x160">
-		<link rel="icon" type="image/png" href="/favicon-96x96.png" sizes="96x96">
-		<link rel="icon" type="image/png" href="/favicon-16x16.png" sizes="16x16">
-		<link rel="icon" type="image/png" href="/favicon-32x32.png" sizes="32x32">
+		<link rel="apple-touch-icon" sizes="57x57" href="{f:uri.resource(path: 'Legacy/icon/apple-touch-icon-57x57.png', package: 'Sfi.Kateheo')}?{f:format.date(date:'now',format:'U')}">
```

Comment:
What do updated favicons have to do with tags functionality? I thought you had it in a separate PR?